_data/accessibility-tests/typography.yml

```diff
@@ -0,0 +1,32 @@
+title: Typography
+component_status: pass
+test_items:
+# General tests
+  - summary: Text meets color contrast requirements.
+    summary_additional: |
+      When you view text on a page and use ANDI or color contrast analyzer to look at the hex colors,
+      the font color meets WCAG color contrast of 4.5:1.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.6.1
+    wcag_criterion: 1.4.3
+  - summary: Headings are present, appropriate and follow logical order.
```

Comment:
add comma after "appropriate"

Reply:
Fixed in [1bc1990](https://github.com/uswds/uswds-site/pull/2540/commits/1bc1990505475ff657e314bd58a8a746577dcf1e)